spec/requests/bulk_upload_controller_spec.rb

```diff
@@ -50,6 +58,40 @@
       end
     end
 
+    describe "GET #start" do
+      before do
+        Timecop.freeze(time)
+        get start_bulk_upload_lettings_logs_path
+      end
+
+      after do
+        Timecop.unfreeze
+      end
+
+      context "when not crossover period" do
```

Comment:
there should be coverage in `spec/features/bulk_upload_lettings_logs_spec.rb` but I have no problems with extra coverage